pkg/pillar/cmd/zedagent/handlenetworkinstance.go

```diff
@@ -761,6 +761,56 @@ func handleAppFlowMonitorDelete(ctxArg interface{}, key string,
 	log.Infof("handleAppFlowMonitorDelete(%s) done\n", key)
 }
 
+func handleAppVifIPTrigModify(ctxArg interface{}, key string,
+	statusArg interface{}) {
+
+	log.Infof("handleAppVifIPTrigModify(%s)\n", key)
+	ctx := ctxArg.(*zedagentContext)
+	trig := cast.CastVifIPTrig(statusArg)
+	if trig.Key() != key {
```

Comment:
You don't need this.. This was obsolete code. Please remove the If block.

Reply:
ok. was copying from someplace.